onnxscript/function_libs/torch_lib/ops/core.py

```diff
@@ -6755,13 +6755,26 @@ def aten_positive(self: TensorType) -> TensorType:
 @torch_op(("aten::pow.Tensor_Tensor", "_operator::pow"), trace_only=True)
 def aten_pow(self: TReal, exponent: TTensor) -> TReal:
     """pow(Tensor self, Tensor exponent) -> Tensor"""
+    # TODO(justinchuby): Add type promotion
     return op.Pow(self, exponent)
 
 
 @torch_op("aten::pow.Tensor_Scalar", trace_only=True)
 def aten_pow_tensor_scalar(self: TReal, exponent: float) -> TReal:
```

Comment:
Add int to exponent

Reply:
The schema parser we have doesn't support this union. Since in python float "includes" int, I did it this way instead.